### PR TITLE
Update for chair emeritus

### PR DESCRIPTION
**Pull request by zacharysarah**
ref: #23797

## 🟢 OK TO MERGE 🟢 

Don't merge this PR until all conditions are true:
- [x] all governance pieces of #23797 are approved
- [x] Date equal to or later than September 15th, 2020

## Description

This PR updates repo permissions and contacts for @irvifa becoming a chair and @zacharysarah transitioning to chair emeritus.

/assign @irvifa

### CODE REVIEW

**Comment by tengqm** [comment]
**SIG Docs Co-Chair prerequisites and responsibilities:**
https://kubernetes.io/docs/contribute/advanced/#serve-as-a-sig-co-chair

File: SECURITY_CONTACTS [hunk 1]
```diff
@@ -10,6 +10,7 @@
 # DO NOT REPORT SECURITY VULNERABILITIES DIRECTLY TO THESE NAMES, FOLLOW THE
 # INSTRUCTIONS AT https://kubernetes.io/security/
 
+irvifa
 jimangel
 kbarnard10
-zacharysarah
+sftim
```
Comment (sftim) [inline SECURITY_CONTACTS:16]: LGTM.

**Comment by jimangel** [comment]
👋 I'm going to be making some permission changes in some of the same files, let me know after you resolve conflicts on Slack and I can approve / fix my PRs 😄. Thanks!

**Comment by zacharysarah** [comment]
@jimangel 👋 Rebase complete!

**Pull request merged**